addons/event_hr_skills/tests/test_event_hr_skills.py

```diff
@@ -0,0 +1,39 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import Command
+from odoo.tests import common
+
+
+class TestEventHrSkills(common.TransactionCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        cls.alice = cls.env['hr.employee'].create({
```

Comment:
Don't use naming based on random naming, use real content naming like
```suggestion
        cls.test_employee = cls.env['hr.employee'].create({
```